### PR TITLE
Added __changeLang route to change fwLocale

### PR DESCRIPTION
**Pull request by skeltonm**
This allows a user to easily switch to view the site in another language

### CODE REVIEW

**Comment by lmajano** [comment]
Ohhhh I like this!! How about if in the CBHelper we add a new method for rendering this link, something like:

```
cb.linkLanguageChange( lang = "en_US" );
```

With a default argument which is the iso code language defaulting to english?

Thoughts?

**Comment by skeltonm** [comment]
Yes, I like that idea.  I added that to CBHelper.  It looks like you changed it from linkLanguageChange to linkLanguage.  Is linkLanguage what you prefer?

File: modules/contentbox/models/system/CBHelper.cfc [hunk 1]
```diff
@@ -1417,6 +1417,14 @@ component accessors="true" singleton threadSafe{
 	}
 
 	/**
+	 * Link to the __changeLang route, this is where the fwLocale is changed
+	 * @lang The iso language code
+	 */
+	function linkLanguageChange( string lang = "en_US" ) {
+		return getRequestContext().buildLink('__changeLang/' & arguments.lang);
```
Comment (lmajano) [inline modules/contentbox/models/system/CBHelper.cfc]: Do you mind adding the Ortus spacing guide to this:

```
return getRequestContext().buildLink( '__changeLang/' & arguments.lang );
```

Comment (skeltonm) [inline modules/contentbox/models/system/CBHelper.cfc]: I added the Ortus spacing guide.  Anything else before this is good?

**Comment by lmajano** [comment]
Thanks so much @skeltonm!! These are great for i18n, keep em coming.  We are actually preparing for the 3.6.0 release, so any other updates are welcomed 👍

**Pull request merged**